tests/foreman/cli/test_computeresource_docker.py

```diff
@@ -71,19 +71,22 @@ def test_positive_upload_image(module_org, target_sat, container_contenthost):
         })
         Docker.container.start({'id': container['id']})
         """
-        container = {'uuid': 'stubbed test'}
+        container = {'uuid': 'test'}
         repo_name = gen_string('alphanumeric').lower()
 
         # Commit a new docker image and verify image was created
         image_name = f'{repo_name}/{CONTAINER_UPSTREAM_NAME}'
+        container_contenthost.execute(
+            f'podman run -d -it --name test docker.io/{CONTAINER_UPSTREAM_NAME}'
```

Comment:
I think better to just specify only container name, and let podman decide where it wants to pull image from, wdyt?
```suggestion
            f'podman run -d -it --name {container["uuid"]} {CONTAINER_UPSTREAM_NAME}'
```